Doc/library/argparse.rst

```diff
@@ -635,6 +635,22 @@ be positional::
    usage: PROG [-h] [-f FOO] bar
    PROG: error: the following arguments are required: bar
 
+You do not need to specify the dest_ and metavar_ parameters.  The
```

Comment:
I might suggest adding one sentence before this for additional clarity and feel a little less abrupt...perhaps something like:

"By default, argparse automatically handles the internal naming and display names of arguments, simplifying the process without requiring additional configuration. As such, you do not need to specify the..."

Reply:
Done.